### PR TITLE
Add kotlin dsl support

### PR DESCRIPTION
**Pull request by devtribe**
This fixes issue #163

### CODE REVIEW

File: examples/single-module-kotlin-dsl-gradle/src/main/java/com/example/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class HelloWorld {
+
+    public static void main(String[] args) {
```
Comment (Reamer) [inline examples/single-module-kotlin-dsl-gradle/src/main/java/com/example/HelloWorld.java]: Hi @devtribe,
thanks for your PR. I think your example is `java`-code. Can you rewrite this to `kotlin`?
Best Regards
Reamer

Comment (devtribe) [reply examples/single-module-kotlin-dsl-gradle/src/main/java/com/example/HelloWorld.java]: Hi @Reamer,

sure i can do this, but the code of the application is not really important, the important thing is the Gradle Kotlin DSL in die `build.gradle.kts`.
Just give me a ping if i should rewrite it.

Greetings

Comment (Reamer) [reply examples/single-module-kotlin-dsl-gradle/src/main/java/com/example/HelloWorld.java]: Yes please rewrite. Our gradle kotlin example application should be written in kotlin.

**Comment by devtribe** [comment]
You're absolutely correct about your changes. Sorry for that, i just build it and don't check if the sonarqube tasks works. Thanks for your open Eye!

**Pull request merged**